### PR TITLE
Modernize Travis

### PR DESCRIPTION
**Pull request by szepeviktor**
One item of #16 
Trying to normalize PHPCompatibility in https://github.com/PHPCompatibility/PHPCompatibility/pull/729

### CODE REVIEW

**Comment by szepeviktor** [comment]
All green 🍏 
only jshint complains https://travis-ci.org/Alecaddd/awps/jobs/415179384

**Comment by Alecaddd** [comment]
This is amazing!
Thank you so much @jrfnl for your thorough review and comments, soooo helpful!
@szepeviktor, you already did a lot, so if you want I can merge and fix everything. No need to spend more time on this 😄

**Comment by szepeviktor** [comment]
> I can merge and fix everything

All right!

File: .gitignore [hunk 1]
```diff
@@ -46,6 +46,10 @@ vendor
 wpcs/
```
[the remark this answers is not in the document]
Comment (Alecaddd) [reply .gitignore]: A leftover for when I tried having WPCS directly in the repo

**Pull request merged**